### PR TITLE
Fix `file:` URIs under Windows

### PR DESCRIPTION
**Pull request by corruptmemory**
This PR will write `file:` URIs under Windows as `file:////` instead of `file://`.  This scheme successfully makes use of the offline repo.

### CODE REVIEW

**Comment by eed3si9n** [comment]
> Do not add (and remove) local local resolvers under Windows #1068

To distinguish with `local` repository (`~/.ivy2/local/`), could we call `activator-local` repository as "preloaded local repository", or "activator-local repository"?

Also to be clear, this fix just removes the `repositories` entry, but the preloaded local repo will still be shipped (for now)?

**Comment by corruptmemory** [comment]
Yes, we're referring to the lines in `~/.sbt/repositories` that look like:

```
activator-launcher-local: file://${activator.local.repository-${activator.home-${user.home}/.activator}/repository}, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext]
activator-local: file://${activator.local.repository-//C:/Users/jim/Desktop/activator-dist-1.3.5/repository}, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext]
```

The preloaded local repo is still shipped, but not used.

**Comment by corruptmemory** [comment]
Hrm.  Going to experiment with this some more.  I think there may be a better fix.  Don't merge yet.

**Comment by corruptmemory** [comment]
OK, this PR has had a major update.  @jsuereth, @eed3si9n please have a look. /cc @henrikengstrom

File: launcher/src/main/scala/activator/RepositoryConfig.scala [hunk 1]
```diff
@@ -112,30 +119,33 @@ object RepositoryConfig {
       old
   }
 
+  private lazy val fileMarker = if (isWindows) "file:////" else "file://"
```
Comment (eed3si9n) [inline launcher/src/main/scala/activator/RepositoryConfig.scala]: Do you have any link to indicate why/how this works? Maybe add a comment for future maintainer?

Comment (corruptmemory) [inline launcher/src/main/scala/activator/RepositoryConfig.scala]: Good point.

So 1) users figured out that `file:///` works.  2) I noticed that the Activator launcher generates log lines with `file:////` in them. and 3) there is this: https://github.com/typesafehub/activator/blob/master/dist/src/templates/activator.bat#L209

So, I think that either `file:///` or `file:////` will work as expected under Windows.

TL;DR: I'll add a comment.
